bundles/org.openhab.binding.yioremote/src/main/resources/ESH-INF/i18n/yioremote_de_DE.properties

```
@@ -0,0 +1,20 @@
+# binding
+binding.yioremote.name = YIOremote Addon
+binding.yioremote.description = Dieses Addon steuert eine YIO Dock / Remote-Kombination
+
+# thing types
+thing-type.yioremote.yioremotedock.label = YIO Remote Dock Thing
+thing-type.yioremote.yioremotedock.description = Stellt das YIO Remote Dock Thing bereit
+
+# thing type config description
+thing-type.config.yioremote.yioremotedock.host.label = IP-Adresse oder Hostname
+thing-type.config.yioremote.yioremotedock.host.description = IP-Adresse oder Hostname des YIO-Docks
```

Can you use the same notation over the whole binding?
```suggestion
thing-type.config.yioremote.yioremotedock.host.description = IP-Adresse oder Hostname des YIO Remote Docks
```

---

resolved